platform/test/resources/TestUtility.js

```diff
@@ -11,3 +14,57 @@ export function configObjectEquals(configA, configB){
 
     return (strConfigA === strConfigB)
 }
+
+// Jasmine custom matchers
+export const customMatchers = {
+    
+    /**
+     * Matcher to check the expected keywords are present in the give input.
+     * where actual is a Sting and expected is an array of strings that contains 
+     * the keyword to check.
+     */
+    toContainKeywords : function(matchersUtil){
+        return {
+            compare: function(actual, expected){
+
+                const result = {
+                };
+
+                // Check the condition
+                let containedResults =  expected.map( (akeyword)=> matchersUtil.contains(actual, akeyword) )
+                result.pass = containedResults.every(r => r);
```

Comment:
How about

```suggestion
                result.pass = expected.every( (akeyword)=> matchersUtil.contains(actual, akeyword) );
```

as a simplification?